### DISCUSSION TITLE
Installing in Linux Mint

### DISCUSSION THREAD

**Discussion by Perkolator** [puddletag/puddletag#718] [Q&A] [answered]
What is the correct way to install in Linux Mint? So that the program can be found in the start menu and e.g. in mouse "open with" context menu.

**Comment by infidelus** [comment, marked as answer]
The version in the Linux Mint (Ubuntu) repository is old.  Either use git clone or download and extract the latest version in 'Releases'.  There is a 'puddletag.desktop' shortcut in the source, but the %F in the Exec didn't work for me so I just right-clicked on the Desktop and created a new launcher.
